### DISCUSSION TITLE
support for a experimental hardware

### DISCUSSION THREAD

**Discussion by rajnish22-glitch** [ExpressLRS/ExpressLRS#1561] [Q&A]
Hi 

i had some hardware at my desk for a long time so i figured to build it, i want to make this hardware opensource as well.
as a contribution to this project .In the future this is also push manufacturers to push the hardware
stack-up.
I have attached the pinout for reference 
kindly let me know ,i have wanted to make this tx since ELRS first came out

in my design the rf chain is also simple
[SCHEMATIC.PDF](https://github.com/ExpressLRS/ExpressLRS/files/8633113/SCHEMATIC.PDF)
[RF SWITCH DATA SHEET.pdf](https://github.com/ExpressLRS/ExpressLRS/files/8633115/RF.SWITCH.DATA.SHEET.pdf)
![RF CHAIN](https://user-images.githubusercontent.com/761
[PINOUT ELRS TX.xlsx](https://github.com/ExpressLRS/ExpressLRS/files/8633306/PINOUT.ELRS.TX.xlsx)
62240/166972277-dc02b6ee-1e12-490d-8a9d-653de6ad4b81.jpeg)
PCB IMAGES
![WhatsApp Image 2022-05-05 at 9 23 14 PM (1)](https://user-images.githubusercontent.com/76162240/166979069-5b51c3eb-df2e-4690-91fc-3078c1c81683.jpeg)

![WhatsApp Image 2022-05-05 at 9 23 13 PM (1)](https://user-images.githubusercontent.com/76162240/166979077-a14767af-9e54-4292-9770-261049d50c24.jpeg)

**Comment by pkendall64** [comment]
My first thoughts are that UART0 on the should only go to the USB. Move TX/RX that go to the 8266 to IO16/17 and add a GPIO to the EN pin and another to the BOOT (GPIO0) of the 8285. That way the backpack 8285 can be updated via serial passthrough. So no switches necessary.

Also, it'd be better if the Schematic was shared on easyeda.com so we could more easily see where connections go.

**Reply by rajnish22-glitch** [reply]
Thanks for the heads-up i would do that but right now my main focus is towards TX
the link to the schematic for your reference is attached

[https://drive.google.com/file/d/1Jqd2tAacWZnThLFWFbrr3tRQb3yAGBaI/view?usp=sharing](url)

Follow this link to know how to import the ASCII format schematic in EASYEDA
[https://docs.easyeda.com/en/Import/Import-Altium-Designer/index.html#Import-Schematic-and-PCB](url)

i will make a final revision of this circuit again as some more changes need to be done
I will make it more readable in terms of what goes where
